drivers/node/aws/aws.go

```diff
@@ -180,6 +180,11 @@ func (a *aws) ShutdownNode(n node.Node, options node.ShutdownNodeOpts) error {
 	return nil
 }
 
+// TODO add AWS implementation for this
+func (a *aws) CheckIfPathExists(path string, n node.Node, options node.ConnectionOpts) (bool, error) {
```

Comment:
I might change this in the current review. It should not matter a lot other that the signature change